### DISCUSSION TITLE
MSG Fails to save

### DISCUSSION THREAD

**Discussion by mdville91** [stella-cv/stella_vslam#616] [Q&A]
I am using Socket to view and run a larger dataset (6GB). I cannot get the MSG file to save and struggling to find a reason why. It looks to kill itself when trying to process that data. 

![image](https://github.com/user-attachments/assets/31a1c45f-44a7-43f7-a290-e7e3d0d4b2a1)

**Comment by Skwangles** [comment]
Try use sqlite3, MSG file is where the computer needs to save all 6GB to memory and then convert it to minimised JSON (msg)

See my duplicate issue:
https://github.com/stella-cv/stella_vslam/discussions/591
